docparse/docparse.go

```diff
@@ -209,14 +209,16 @@ func parseComment(prog *Program, comment, pkgPath, filePath string) ([]*Endpoint
 	// Remove startlines and tagline from comment.
 	comment = strings.TrimSpace(comment[start+i:])
 
+	parsingDesc := false
 	pastDesc := false
 	var err error
 
 	// Get description and Kommentaar directives.
 	for _, line := range strings.Split(comment, "\n") {
 		i++
 
-		if strings.TrimSpace(line) == "" {
+		// Ignore empty lines unless we're parsing the description because it needs to include them
+		if strings.TrimSpace(line) == "" && !parsingDesc {
```

Comment:
Yeah I just wanted to be safe because I didn't know the code and it less likely to break in futures if someone changes something else. But no prob, will update